test/integration/targets/azure_rm_networkinterface/tasks/main.yml

```diff
@@ -1,29 +1,34 @@
 - name: Prepare random number
   set_fact:
     rpfx: "{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
+    auth_source: auto
```

Comment:
`auth_source` should be inherit from `azure_rm_common` with `auto` value, no need write at this place

Reply:
I have these in my tests as I authenticate with `auth_source: cli` and change it back to `auto` before pushing. I could not find a common file for tests. I would be super happy to remove if there is a way to easily set this for testing. Using the other auth methods failed for me.